SPECS/k3s/k3s.spec

```diff
@@ -0,0 +1,67 @@
+%global debug_package %{nil}
+%define install_path  /usr/local/bin
+%define util_path     %{_datadir}/k3s
+%define install_sh    %{util_path}/setup/install.sh
+%define uninstall_sh  %{util_path}/setup/uninstall.sh
+%define k3s_binary    k3s
+
+Name:    k3s
+Version: 1.23.6
```

Comment:
K3s is using the version string convention like `v1.23.6+k3s1`. Do we need to be aligned with it?

Reply:
I chose this format for version so that it could be used easily in the rest of the spec file such as:
`%{name}-%{version}-k3s1.tar.gz` or `Release: %{version}%{?dist}`

I also searched through the spec files on CBL-Mariner and didn't find any non-numeric versions. @Camelron Is there precedent for that?

Reply:
Looking at the releases on github (https://github.com/k3s-io/k3s/releases), it seems like the k3s1 string doesn't really differentiate between source versions. What we could do instead is rename the tar to the standard %{name}-%{version}.tar.gz. 

An example of how to do that is here: https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/jflex/jflex.spec#L27